x-pack/plugins/cases/public/components/custom_fields/text/edit.tsx

```diff
@@ -162,7 +162,7 @@ const EditComponent: CustomFieldType<CaseCustomFieldText>['Edit'] = ({
         direction="column"
       >
         {!isCustomFieldValueDefined && !isEdit && (
-          <p data-test-subj="no-tags">{NO_CUSTOM_FIELD_SET(customFieldConfiguration.label)}</p>
+          <p data-test-subj="no-tags">{NO_CUSTOM_FIELD_SET}</p>
```

Comment:
My mistake on a previous PR but could you please change the `no-tags` to `no-custom-field-value`?